theme/forms.py

```diff
@@ -265,6 +266,27 @@ class Meta:
     password1 = forms.CharField(label="Password", widget=forms.PasswordInput())
     password2 = forms.CharField(label="Confirm Password", widget=forms.PasswordInput())
 
+    challenge = forms.CharField()
+    response = forms.CharField()
+
+    def __init__(self, request, *args, **kwargs):
+        self.request = request
+        super(SignupForm, self).__init__(*args, **kwargs)
+
+    def verify_captcha(self):
+        params = dict(self.cleaned_data)
+        params['privatekey'] = getattr(settings, 'RECAPTCHA_PRIVATE_KEY', "6LdNC_USAAAAADNdzytMK2-qmDCzJcgybFkw8Z5x")
+        params['remoteip'] = self.request.META['REMOTE_ADDR']
+        resp = requests.post('http://www.google.com/recaptcha/api/verify', params=params)
```

Comment:
Only using `http` will likely cause an issue when using SSL certs.

Either need to update the `http` / `https` sed statement in hsctl or set it to use either.